### DISCUSSION TITLE
OnUpdate only after candle close?

### DISCUSSION THREAD

**Discussion by LuanG01** [Mathieu2301/TradingView-API#99] [Q&A] [answered]
Hey,
is there any function that will only update chart or indicator information after the candle of the information specified in my setMarket closed?
The .onUpdate function triggers anytime the price changes.
But for my strategy I need all information of the last candle close only.

Thanks in advance!

**Comment by LuanG01** [comment]
You could close this topic as Ive found a workaround. Would still be cool to have a built-in function for this.

**Reply by brunowego** [reply]
@LuanG01 can you share your workaround? Thanks.

**Comment by Mathieu2301** [comment, marked as answer]
> Would still be cool to have a built-in function for this.

Thank you for the suggestion, I take note.
